k8s/secrets.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: caesarapp-secret
+  namespace: default
+data:
+  ## https://symfony.com/doc/current/reference/configuration/framework.html#secret
+  APP_SECRET: < Its value should be a series of characters, numbers and symbols chosen randomly and the recommended length is around 32 characters. >
+  BACKUP_CODE_HASH_LENGTH: <0-9*>
+  BACKUP_CODE_SALT: < Its value should be a series of characters, numbers and symbols chosen randomly and the recommended length is around 32 characters. >
+  ## https://symfony.com/doc/current/doctrine.html#configuring-the-database
+  DATABASE_URL: < pgsql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB} >
+  ## https://developers.google.com/identity/sign-in/web/sign-in
+  GOOGLE_ID: < YOUR_CLIENT_ID.apps.googleusercontent.com >
+  GOOGLE_SECRET: < your google secret >
+  INVITATION_SALT: < Its value should be a series of characters, numbers and symbols chosen randomly and the recommended length is around 32 characters. >
+  JWT_PASSPHRASE: < Its value should be a series of characters, numbers and symbols chosen randomly and the recommended length is around 32 characters. >
+  ## https://symfony.com/doc/current/email.html#configuration
+  MAILER_URL: < smtp://localhost:25?encryption=ssl&auth_mode=login&username=&password= >
+  ## < Rudiment which must be removed! >
+  ALLOW_FRONT_REDIRECT_PATTERN: < /^http[s]*:\/\/(example.com|example2.com)/ > 
+  SUPERADMIN_PASS: < Rudiment which must be removed! >
```

Comment:
@mg-4xxi @beshkenadze Please pay attention to these variables.
`ALLOW_FRONT_REDIRECT_PATTERN` and `SUPERADMIN_PASS`